addons/info/README.md

```diff
@@ -72,6 +72,57 @@ setDefaults({
 setAddon(infoAddon);
 ```
 
+### React Docgen & Flowtype
+
+Install the babel docgen plugin
+
+```bash
+npm install -D babel-plugin-react-docgen
+```
+
+Add the plugin to your `.babelrc` config
+
+```json
+{
+  "plugins": ["react-docgen"]
+}
+```
+
+And add the flowtype definitions to your component
+
+```js
+// @flow
+import React from 'react';
+
+type PropsType = {
```

Comment:
does this mean flowtype is supported by our `babel-plugin-react-docgen`?

https://github.com/storybooks/babel-plugin-react-docgen/issues/29